noconflict.csx

```diff
@@ -0,0 +1,181 @@
+                InvocationExpression(
+                    MemberAccessExpression(
+                        SyntaxKind.SimpleMemberAccessExpression,
+                        IdentifierName("MoreEnumerable"),
+                        IdentifierName(md.Identifier)),
+                    ArgumentList(SeparatedList<ArgumentSyntax>(
+                        from p in md.ParameterList.Parameters
+                        select Argument(IdentifierName(p.Identifier)))))
+                    .WithLeadingTrivia(Space)
+            select
+                MethodDeclaration(md.ReturnType, md.Identifier)
+                    .WithAttributeLists(md.AttributeLists)
+                    .WithModifiers(md.Modifiers)
+                    .WithTypeParameterList(md.TypeParameterList)
+                    .WithConstraintClauses(md.ConstraintClauses)
+                    .WithParameterList(md.ParameterList)
+                    .WithBody(
+                        // TODO Replace with expression-bodied
+                        Block(md.ReturnType.WithoutTrivia().IsEquivalentTo(@void)
+                              ? (StatementSyntax)ExpressionStatement(call)
+                              : ReturnStatement(call))
+                            .WithOpenBraceToken(ParseToken("{").WithLeadingTrivia(Whitespace(indent2))
+                                                               .WithTrailingTrivia(Whitespace(lf + indent3)))
+                            .WithCloseBraceToken(ParseToken("}").WithLeadingTrivia(Whitespace(" " + lf + indent2))))
+    }
+    into m
+    select $@"
+    /// <summary><c>{m.Name}</c> extension.</summary>
+    public static class {m.Name}Extension
+    {{
+{string.Join(null, from mo in m.Overloads select mo.ToFullString())}
+    }}";
+
+var template = $@"
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (C) 2008 Jonathan Skeet.
+// Portions Copyright (C) 2009 Atif Aziz, Chris Ammerman, Konrad Rudolph.
+// Portions Copyright (C) 2010 Johannes Rudolph, Leopold Bushkin.
+// Portions Copyright (C) 2015 Felipe Sateler, ""sholland"".
+// Portions Copyright (C) 2016 Leandro F. Vieira (leandromoh).
+// Portions Copyright (C) Microsoft. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the ""License"");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an ""AS IS"" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+// This code was generated by a tool. Any changes made manually will be lost
+// the next time this code is regenerated.
+// Generated: {DateTimeOffset.Now.ToString("r")}
```

Comment:
I wonder if this Generated header is really necessary. It introduces git noise everytime the tool is run, even if nothing else changed.